tests/test_utils/tests.py

```diff
@@ -4,13 +4,13 @@
 import unittest
 import warnings
 
+from django import forms
```

Comment:
Please keep cleanups like this in a separate commit or pull request to avoid blaming those changes to a commit which contains new features.